composeApp/src/androidMain/res/values/strings.xml

```diff
@@ -792,4 +792,5 @@
     <string name="repeat">Repeat</string>
     <string name="original_lyrics">Original Lyrics</string>
     <string name="translated_lyrics">Translated Lyrics</string>
+    <string name="edit_description">Edit Playlist Description</string>
```

Comment:
I think it should be "edit_playlist_description" because of potential artist page description editing.

Reply:
Thanks for pointing that idea. Changed it in new commit https://github.com/fast4x/RiMusic/pull/4586/commits/44344722af19569c2a1d21fe8d6226e57cb56130